website/public/js/app.js

```diff
@@ -150,12 +150,17 @@ window.habitrpg = angular.module('habitrpg',
           url: '/:gid',
           templateUrl: 'partials/options.social.guilds.detail.html',
           title: env.t('titleGuilds'),
-          controller: ['$scope', 'Groups', 'Chat', '$stateParams',
-          function($scope, Groups, Chat, $stateParams){
-            Groups.Group.get({gid:$stateParams.gid}, function(group){
-              $scope.group = group;
-              Chat.seenMessage(group._id);
-            });
+          controller: ['$scope', 'Groups', 'Chat', '$stateParams', 'Members',
+          function($scope, Groups, Chat, $stateParams, Members){
+            Groups.Group.get($stateParams.gid)
+              .then(function (response) {
+                $scope.group = response.data.data;
+                Chat.seenMessage($scope.group._id);
+                Members.getGroupMembers($scope.group._id)
```

Comment:
In the members box, if you scroll down there's a list of invited user: 
<img width="471" alt="schermata 2016-04-25 alle 21 34 17" src="https://cloud.githubusercontent.com/assets/589911/14796423/8a87dfde-0b2d-11e6-8661-0bfc465ebadc.png">